docs/documentation/siddhi-4.0.md

```diff
@@ -954,18 +945,18 @@ from (every)? <event reference>=<input stream>[<filter condition>] ->
 select <event reference>.<attribute name>, <event reference>.<attribute name>, ...
 insert into <output stream>
 ```
-| Items | Description |
+| Items| Description |
 |-------------------|-------------|
-| `->` | This represent followed by, i.e. After a event matching the first condition the next event matching the next condition arriving sometime after the first event (not necessarily need to be immediately after the previous event event) |
-| `<event reference>` | Let you add a reference to the the matching event such that it can be accessed later for further processing. |
-| `(within <time gap>)?` | `within` clause is optional, It defines the the duration within all the matching events should occur |
-| `every` | `every` is an optional keyword. This defines weather the event matching should be triggered for every event arrival for the specified stream with the matching condition. <br/> When this keyword is not used the matching happens only ones. |
+| `->` | This is used to indicate what the event is followed by. After defining the condition the first event should meet, add this item and then define the condition to be met by the event that follows the first event. Note that the event that should follow need not occur immediately after the first event. |
```

Comment:
This is wrong please fix

Reply:
Is this revision correct?

 "This is used to indicate an event that should be following another event during the time interval specified via the `within` clause. The second event does not necessarily have to occur immediately after the second event. The condition to be met by the first event should be added before the sign, and the condition to be met by the second event should be added after the sign."